### ISSUE TITLE
githubでの作業における注意点

### ISSUE THREAD

**Issue by couunderbarz** [shotanaka0513/gunosy#2]
- 1つのissueに対して1つのbranchを対応させて作業する
- masterブランチへ直接のコミットはしない
- commitは作業単位にまとめてわかりやすいコミットメッセージを付ける
- PRには対応するissueを紐付ける
- issueに対する質問はissueに(例えばgithubの使い方の質問はこちらに書いてください)

**Comment by shotanaka0513** [comment]
吉田様
いつもお世話になっております。データ分析アルバイトの課題修正中の田中と申します。
お忙しい中申し訳ありませんが１点ほど質問があり、連絡いたしました。
このgithubでの作業における注意点 #2なのですが、こちらのissueは注意点であるためbranchを作成してPRを作成する必要が無いと判断したのですが、いかがでしょうか？
返信よろしくお願いいたします。

田中

**Comment by couunderbarz** [comment]
はい。このissue用のプルリクは作成必要ありません。他のissueをcloseした後に、このissueもcloseするようにしますmm

**Comment by shotanaka0513** [comment]
吉田様
連絡ありがとうございます。承知いたしました。

田中

**Comment by couunderbarz** [comment]
LGTMついたプルリクは田中さんの方でマージするようお願いしますmm
（コメントに `吉田様` `いつもお世話になっております。` `田中` 等は無くても大丈夫ですよ。）

**Comment by shotanaka0513** [comment]
承知いたしました。

**Comment by shotanaka0513** [comment]
LGTMのついてプルリクをマージ致しました。マージしたissueはこちら側でcloseし、それに対応するbranchを消してもよろしいでしょうか？

**Comment by shotanaka0513** [comment]
修正中のrequirements.txtの追加(#7)が終了致しました。お時間要しまして申し訳ありませんでした。確認お願いいたします。

**Comment by couunderbarz** [comment]
> shotanaka0513 commented an hour ago
> LGTMのついてプルリクをマージ致しました。マージしたissueはこちら側でcloseし、それに対応するbranchを消してもよろしいでしょうか？

はい。消して大丈夫です 👍

**Comment by shotanaka0513** [comment]
承知致しました。ありがとうございます。

**Comment by shotanaka0513** [comment]
また夜分での対応ありがとうございます。
